mcp/motors/actuators.c

```diff
@@ -55,11 +53,10 @@ extern int16_t InCharge;		/* tx.c */
 
 /* Index for each stepper for structures, name, id */
 #define LOCKNUM 4
-#define HWPRNUM 5
 #define SHUTTERNUM 6
 static const char *name[NACT] = {"Actuator #0", "Actuator #1", "Actuator #2",
-				 "Balance Motor", "Lock Motor", HWPR_NAME, "Shutter", "Pot Valve",
-				 "Pump 1 Valve", "Pump 2 Valve"};
+				 "Balance Motor", "Lock Motor", "FREE_1", "Shutter", "FREE_2",
```

Comment:
Comment that FREE_N are placeholders and available for use.

Reply:
Added